### PR TITLE
add-page-views-to-audience

### PR DESCRIPTION
**Pull request by mes1901**
https://trello.com/c/EChDghOS

### CODE REVIEW

File: frontend/src/components/dashboard/page_views/ButtonComponent.vue [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<template>
```
Comment (TatsuUkraine) [inline frontend/src/components/dashboard/page_views/ButtonComponent.vue]: don't copy/paste, create 1 common component, that will work with just props, use it here and in visitors page

File: frontend/src/components/dashboard/page_views/PeriodDropdown.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<template>
```
Comment (TatsuUkraine) [inline frontend/src/components/dashboard/page_views/PeriodDropdown.vue]: don't copy/paste - make 1 common component that will be working with just props, and use it here and in Visitors page

File: frontend/src/pages/PageViews.vue [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    import {isWebsite} from '../mixins/isWebsite';
+    import {
+        PAGE_VIEWS,
+        UNIQUE_PAGE_VIEWS,
+        ACTIVE_USERS,
+        BOUNCE_RATE
+    } from '../configs/page_views/buttonTypes.js';
+
+    export default {
+        mixins: [isWebsite],
```
Comment (TatsuUkraine) [inline frontend/src/pages/PageViews.vue]: don't put guard on each child page, you have 1 page that wraps all of them, which means that guard mixin should be used only there (it also means that Visitors page no longer need it too)

**Comment by TatsuUkraine** [comment]
@mes1901 reach me you have any questions

File: frontend/src/components/dashboard/common/PeriodDropdown.vue [hunk 4]
```diff
@@ -9,44 +9,38 @@
...
+        },
         data() {
             return {
                 items: [
                     {title: 'Today', value: 'today'},
                     {title: 'Yesterday', value: 'yesterday'},
                     {title: 'Last 7 days', value: 'last_week'},
-                    {title: 'Last month', value: 'last_month'},
-                    {title: 'Last quartal', value: 'last_quartal'},
+                    {title: 'Last 30', value: 'last_month'},
```
Comment (lenchvolodymyr) [inline frontend/src/components/dashboard/common/PeriodDropdown.vue]: it was okay

**Pull request merged**